ssr-testing/pages/select.tsx

```diff
@@ -29,30 +29,32 @@ export default function SelectPage() {
         </Select.Content>
       </Select.Root>
 
-      <Select.Root>
-        <Select.Trigger>
-          <Select.Value placeholder="Pick an option" />
-          <Select.Icon>▼</Select.Icon>
-        </Select.Trigger>
-        <Select.Content>
-          <Select.ScrollUpButton>▲</Select.ScrollUpButton>
-          <Select.Viewport>
-            <Select.Item value="1">
-              <Select.ItemText>Item 1</Select.ItemText>
-              <Select.ItemIndicator>✔</Select.ItemIndicator>
-            </Select.Item>
-            <Select.Item value="2">
-              <Select.ItemText>Item 2</Select.ItemText>
-              <Select.ItemIndicator>✔</Select.ItemIndicator>
-            </Select.Item>
-            <Select.Item value="3">
-              <Select.ItemText>Item 3</Select.ItemText>
-              <Select.ItemIndicator>✔</Select.ItemIndicator>
-            </Select.Item>
-          </Select.Viewport>
-          <Select.ScrollDownButton>▼</Select.ScrollDownButton>
-        </Select.Content>
-      </Select.Root>
+      <form>
```

Comment:
Do you think there's value in rendering both cases for quick manual testing?

Reply:
> I just thought we should have one with and one without a form

Ah! that's exactly what I was asking for before I took a second to expand the diff and saw that's what's been done here 😅 

All good!